yt_dlp/extractor/nitter.py

```diff
@@ -168,61 +231,51 @@ def _real_extract(self, url):
                 'ext': ext
             }]
 
-        title = self._og_search_description(full_webpage)
-        if not title:
-            title = self._html_search_regex(r'<div class="tweet-content[^>]+>([^<]+)</div>', webpage, 'title')
+        title = self._og_search_description(full_webpage) or self._html_search_regex(
+            r'<div class="tweet-content[^>]+>([^<]+)</div>', webpage, 'title', fatal=False)
         description = title
 
-        mobj = self._match_valid_url(url)
-        uploader_id = (
-            mobj.group('uploader_id')
-            or self._html_search_regex(r'<a class="fullname"[^>]+title="([^"]+)"', webpage, 'uploader name', fatal=False)
-        )
+        uploader_id = (self._html_search_regex(
+            r'<a class="username"[^>]+title="@([^"]+)"', webpage, 'uploader id', fatal=False)
+            or self._match_valid_url(url).group('uploader_id'))
```

Comment:
```suggestion
        uploader_id = (
            self._html_search_regex(
                r'<a class="username"[^>]+title="@([^"]+)"', webpage, 'uploader id', fatal=False)
            or self._match_valid_url(url).group('uploader_id'))
```
weird indent. I'm surprised flake8 allows this